frame/assets/src/lib.rs

```diff
@@ -241,7 +241,8 @@ pub mod pallet {
 			+ Copy
 			+ MaybeSerializeDeserialize
 			+ MaxEncodedLen
-			+ TypeInfo;
+			+ TypeInfo
+			+ FixedPointOperand;
```

Comment:
```suggestion
			+ TypeInfo;
```

Reply:
This should not be needed. `FixedPointOperand` has a blanket impl and all bounds required for it are already bounded by `Copy + AtLeast32BitUnsigned`.